src/components/UI/Card/ProductCards/ProductCards.jsx

```diff
@@ -11,99 +18,169 @@ import "./ProductsCards.css";
+                        <span className="text-linethrough discount-price">₹{discountedPrice}</span>
+                    </div>
                 </div>
-                <div className="card-title">
-                    <h2>₹{price}</h2>
-                    <span className="text-grey">{discount}</span>
-                    <span className="text-linethrough discount-price">₹{discountedPrice}</span>
-                </div>
-            </div>
-            <div className="card-footer">
-                <div className="card-footer-view">
-                      <button onClick={() => { AddProductsInCartHandler(props) }}>Add to Cart
-                      <span class="material-icons-round">
-                        shopping_cart
-                        </span>
+                <div className="card-footer">
+                    <div className="card-footer-view">
+                        <button className='btn-addToCart'>
+                            {
+                            cartState.some((cartitem) => cartitem._id === _id) ? <div className='cart-quantity-container'>
+                                <span className={
+                                        `${handleCartQuantity}`
+                                    }
+                                    onClick={
+                                        () => UpdateCartHandler("increment")
+                                }><IcRoundPlus/></span>
+                                {
+                                {
+                                    ...cartState.find((cartitem) => cartitem._id === _id)
+                                } ?. qty
+                            }
+                                <span onClick={
+                                    () => UpdateCartHandler("decrement")
+                                }><IcRoundMinus/></span>
+                            </div> : <span onClick={
+                                    () => {
+                                        AddProductsInCartHandler(props)
+                                    }
+                                }
+                                className="add-cart-action">
+                                <span className='hide'>Add to Cart</span>
+                                <span className="material-icons-round icons-style">
+                                    add_shopping_cart
+                                </span>
 
-                        </button>
+                            </span>
+                        } </button>
+                    </div>
                 </div>
-            </div>
-            <span className="material-icons-round badge topright-badge " onClick={()=>{AddProductsInWishListHandler(props)}}>
-                favorite_border
-            </span>
-        </div>
-    </>
-  )
+                {
+                WishListState.find((cartitem) => cartitem._id === _id) ? <></> : <span className="material-icons-round badge topright-badge badge-border"
+                    style={
```

Comment:
Instead of returning react fragment, you can return null here. Also, instead of ternary operator, a simple && would be a better fit here.

Reply:
okay will update to && operator